src/nl/hannahsten/texifyidea/editor/TablePasteProvider.kt

```diff
@@ -35,6 +35,7 @@ open class TablePasteProvider : PasteProvider {
     }
 
     override fun isPastePossible(dataContext: DataContext): Boolean {
+        return false
```

Comment:
Why?

Reply:
Temporary so I can see how pandoc handles it

Reply:
Ah okay, well let me know if you want me to review the PR again!